<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="contextMenuPolicy">
           <enum>Qt::ActionsContextMenu</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QVTKOpenGLNativeWidget" name="vtkWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>200</horstretch>
            <verstretch>100</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Item Visibility</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Color Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;
font: 700 9pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Model Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionactionItem_Options"/>
    <addaction name="actionactionstartVR"/>
    <addaction name="actionStopVR"/>
    <addaction name="actionRename_File"/>
   </widget>
   <addaction name="menuOpen_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionOpen_Folder"/>
   <addaction name="actionactionItem_Options"/>
   <addaction name="actionactionstartVR"/>
   <addaction name="actionStopVR"/>
  </widget>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionactionItem_Options">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/itemOptions.png</normaloff>:/Icons/itemOptions.png</iconset>
   </property>
   <property name="text">
    <string>Item Options</string>
   </property>
   <property name="toolTip">
    <string>Item Options</string>
   </property>
  </action>
  <action name="actionactionstartVR">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/startVR.png</normaloff>:/Icons/startVR.png</iconset>
   </property>
   <property name="text">
    <string>actionstartVR</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/openFolder.png</normaloff>:/Icons/openFolder.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionStopVR">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/stopVR.png</normaloff>:/Icons/stopVR.png</iconset>
   </property>
   <property name="text">
    <string>StopVR</string>
   </property>
  </action>
  <action name="actionRename_File">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/renameFile.png</normaloff>:/Icons/renameFile.png</iconset>
   </property>
   <property name="text">
    <string>Rename Item</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
